### ISSUE TITLE
Manual `window.render()` is not working on Windows

### ISSUE THREAD

**Issue by Meakk** [f3d-app/f3d#1020]
**Describe the bug**
When running a Python script, it's not possible to render the window manually on Windows and we get a result only when the interactor is started. `render_to_image().save('...')` works as intended though.

**To Reproduce**
```py
import f3d
e=f3d.Engine()
e.window.render()
```

**Expected behavior**
The window should contain the expected render.

**System Information:**
 - OS: Windows

**F3D Information**
2.2 RC2

**Comment by mwestphal** [comment]
It is the same issue has one of these imo: #1004 , we would need a dedicated thread to handle rendering in the window, it is because the OS is not able to talk with the window and think it is not responding. Out of scope of 2.2 release at this point.

**Comment by Meakk** [comment]
I agree about the dedicated thread. It could also solve the issue of the blocking `interactor.start()` too.
